plutus-use-cases/test/Spec/Prism.hs

```diff
@@ -88,3 +116,108 @@ prismTrace = do
     _ <- Trace.waitNSlots 2 -- needed?
     Trace.callEndpoint @"credential manager" uhandle (Trace.chInstanceId chandle)
     void $ Trace.waitNSlots 2
+
+-- * QuickCheck model
+
+data STOState = STOReady | STOPending | STODone
+    deriving (Eq, Ord, Show)
+
+data IssueState = NoIssue | Revoked | Issued
+    deriving (Eq, Ord, Show)
+
+data PrismModel = PrismModel
+    { _walletState :: Map Wallet (IssueState, STOState)
+    }
+    deriving (Show)
+
+makeLenses 'PrismModel
+
+_fromJust :: Lens' (Maybe a) a
+_fromJust f (Just x) = Just <$> f x
+_fromJust f Nothing  = error "_fromJust: Nothing"
+
+isIssued :: Wallet -> Lens' PrismModel IssueState
+isIssued w = walletState . at w . _fromJust . _1
```

Comment:
could you use something like `walletState . at w . non (NoIssue, STOPending)` ? (if that's what the default values are)